src/components/results/feedback/index.js

```diff
@@ -0,0 +1,34 @@
+import {feedbackModalShowState} from "lib/recoil";
+import useLoadedValue from "lib/hooks/use-loaded-value";
+import useTranslate from "lib/hooks/use-translate";
+import {useRecoilState} from "recoil";
+import style from "./style.scss";
+import Modal from "./modal";
+import {userCompletedFeedbackQuery} from "../../../lib/recoil/feedback";
+
+export default function Feedback() {
+  const [show, setShow] = useRecoilState(feedbackModalShowState);
```

Comment:
I think the feedback modal show state can just be a local state handler. The career version was a bit different and needed to be accessed in a different way